pilot/logger/logger.py

```diff
@@ -8,7 +8,11 @@ def setup_logger():
     log_format = "%(asctime)s [%(filename)s:%(lineno)s - %(funcName)20s() ] %(levelname)s: %(message)s"
 
     # Create a log handler for file output
-    file_handler = logging.FileHandler(filename=os.path.join(os.path.dirname(__file__), 'debug.log'), mode='w')
+    file_handler = logging.FileHandler(
+        filename=os.path.join(os.path.dirname(__file__), 'debug.log'),
+        mode='w',
+        encoding='utf-8',
```

Comment:
Adding encoding so logging doesn't crash on Windows when there are characters which don't fit into whatever code page is active by default.